dissect/target/tools/qfind.py

```diff
@@ -40,19 +53,69 @@ def main() -> int:
         log.error("No targets provided")
         return 1
 
+    if args.record or args.json:
+        RECORD_OUTPUT = True
+        rs = record_output(args.strings, args.json)
```

Comment:
Rather initialize `rs = None` and do that `if` later based on that.

Reply:
Why would you need a record stream for each target? Your current code also only has a single record stream.